remote_execution/oss/re_grpc/src/client.rs

```diff
@@ -461,22 +415,22 @@ impl REClient {
 
     pub async fn upload_blob(
         &self,
-        blob: Vec<u8>,
-        metadata: RemoteExecutionMetadata,
+        _blob: Vec<u8>,
+        _metadata: RemoteExecutionMetadata,
     ) -> anyhow::Result<TDigest> {
-        stub("upload_blob")
+        // TODO(aloiscochard)
+        Err(anyhow::anyhow!("Not implemented"))
```

Comment:
Say "RE upload_blob" in the message somewhere, so when it fails this way, we know what to fix.